example/javascriptlib/basic/3-custom-build-logic/build.mill

```diff
@@ -0,0 +1,64 @@
+package build
+
+import mill._, javascriptlib._
+
+trait CustomModule extends JestModule {
```

Comment:
We should not have a `trait` here and just do things directly in `object foo`, for consistency with the other examples